includes/User.php

```diff
@@ -2741,25 +2715,11 @@ public function setLocalAttribute($attribute, $value, $cityId = null, $sep = '-'
 	 */
 	public function setGlobalAttribute($attribute, $value) {
 		$this->setOptionHelper($attribute, $value);
-		$this->setAttributeInService($attribute, $value);
+		$this->markAttributesToSetInService( $attribute );
```

Comment:
The `$this->setOptionHelper` call above will set the attribute in the general `mOptions` array which also contains preferences and flags. Later on when it comes time to actually save these options, we want to know which options were set with this `User::setGlobalAttribute`method. Those options are attributes which we'll then save to the service. See [here](https://github.com/Wikia/app/pull/8216/files#diff-897fba1dd6b8edc3238762e35d0b7c71R5074)